packages/liveblocks-core/src/room.ts

```diff
@@ -3244,56 +3273,97 @@ export function createRoom<
     if (abortSignal?.aborted) {
       throw abortError;
     }
+    setupFileUpload(attachment, options);
```

Comment:
```suggestion
    setupFileUpload(attachment, options);
```

make this noop so `uploadAttachment` is the only thing that's exposed, it would also be called for uploadAttachment